vm/src/stdlib/binascii.rs

```diff
@@ -177,31 +177,113 @@ mod decl {
         encoded
     }
 
+    #[inline]
+    fn uu_a2b_read(s: &[u8], index: usize, vm: &VirtualMachine) -> PyResult<u8> {
+        if let Some(c) = s.get(index) {
+            // Check the character for legality
+            // The 64 instead of the expected 63 is because
+            // there are a few uuencodes out there that use
+            // '`' as zero instead of space.
+            if *c < 32 || *c > 96 {
```

Comment:
```suggestion
            if !(0x20..=0x60).contains(*c) {
```

Hexadecimal literals are used below but not here